extensions/amp-story/1.0/amp-story.js

```diff
@@ -2237,13 +2240,32 @@ export class AmpStory extends AMP.BaseElement {
 
     const pagesByDistance = this.getPagesByDistance_();
 
-    this.mutateElement(() => {
+    const preloadAllPages = () => {
       pagesByDistance.forEach((pageIds, distance) => {
         pageIds.forEach((pageId) => {
           const page = this.getPageById(pageId);
           page.setDistance(distance);
         });
       });
+    };
+
+    this.mutateElement(() => {
+      if (
+        !isExperimentOn(this.win, 'amp-story-load-first-page-only') ||
```

Comment:
Plz report this experiment to the CSI pipeline so we can use it to filter our dashboards
`Services.performanceFor(win).addEnabledExperiment('amp-story-load-first-page-only');`